src/odig.mli

```diff
@@ -0,0 +1,11 @@
+(** Provide an interface to Odig. *)
+
+type t
+
+val of_sandbox : Sandbox.t -> (t, string) result Promise.t
+
+(** Generates the odoc API documentation and manual of [package_name]. Interface
+    to the [odig odoc package_name] command *)
```

Comment:
The return value of this function isn't clear. Could you describe it in the documentation string?